contrib/jenkins/install_catalog.sh

```diff
@@ -22,62 +22,58 @@ ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
 
 while [[ $# -gt 0 ]]; do
   case "${1}" in
-    --registry)               REGISTRY="${2:-}"; shift ;;
-    --version)                VERSION="${2:-}"; shift ;;
-    --release-name)           CATALOG_RELEASE="${2:-}"; shift ;;
-    --service-catalog-config) SC_KUBECONFIG="${2:-}"; shift ;;
-    --with-tpr)               WITH_TPR=1 ;;
-    --fix-auth)               FIX_CONFIGMAP=1 ;;
+    --registry) REGISTRY="${2:-}"; shift ;;
+    --version)  VERSION="${2:-}"; shift ;;
+    --with-tpr) WITH_TPR=true ;;
+    --fix-auth) FIX_CONFIGMAP=true ;;
     *) error_exit "Unrecognized command line parameter: $1" ;;
   esac
   shift
 done
 
-CATALOG_RELEASE="${CATALOG_RELEASE:-"catalog"}"
-
-K8S_KUBECONFIG="${KUBECONFIG:-~/.kube/config}"
-SC_KUBECONFIG="${SC_KUBECONFIG:-"/tmp/sc-kubeconfig"}"
-
-VERSION="${VERSION:-"canary"}"
 REGISTRY="${REGISTRY:-}"
+VERSION="${VERSION:-"canary"}"
+WITH_TPR="${WITH_TPR:-false}"
+FIX_CONFIGMAP="${FIX_CONFIGMAP:-false}"
+
 CONTROLLER_MANAGER_IMAGE="${REGISTRY}controller-manager:${VERSION}"
 APISERVER_IMAGE="${REGISTRY}apiserver:${VERSION}"
 
-echo 'INSTALLING CATALOG'
+echo 'INSTALLING SERVICE CATALOG'
 echo '-------------------'
-echo "Using kubeconfig: ${K8S_KUBECONFIG}"
-echo "Using service catalog kubeconfig: ${SC_KUBECONFIG}"
 echo "Using controller-manager image: ${CONTROLLER_MANAGER_IMAGE}"
 echo "Using apiserver image: ${APISERVER_IMAGE}"
 echo '-------------------'
 
 # Deploying to cluster
 
-retry -n 10 \
+echo 'Deploying service catalog...'
+
+retry \
     kubectl --namespace kube-system get configmap extension-apiserver-authentication \
   || error_exit 'Timed out waiting for extension-apiserver-authentication configmap to come up.'
 
 # The API server automatically provisions the configmap, but we need it to contain the requestheader CA as well,
```

Comment:
Nit, and totally fine for a follow-up.  It's not super-clear what 'the configmap' is.